core/embed/rust/src/protobuf/encode.rs

```diff
@@ -231,6 +230,6 @@ impl<'a> OutputStream for BufferStream<'a> {
                 *pos += 1;
                 *buf = val;
             })
-            .ok_or(Error::Missing)
+            .ok_or(error::end_of_buffer())
```

Comment:
`ok_or_else`

Reply:
fixed in b406bd0a0ff986392c2fccef336a7b9201734023